bigquery/views/job-alert.sql

```diff
@@ -2,6 +2,14 @@ SELECT
   id,
   CAST(created_at AS date) AS created_date,
   CAST(updated_at AS date) AS updated_date,
+  active,
+  CAST(unsubscribed_at AS date) AS unsubscribed_date,
+IF
+  (NOT active,
+    DATETIME_DIFF(unsubscribed_at,
+      created_at,
+      DAY),
+    NULL) AS days_active,
```

Comment:
Re datetime, perhaps this will come out as a strange number like 14.238483 - also don't know if that matters, if true.

Reply:
Interesting thought! I just checked the documentation though, and it should come out as a whole number of DAY intervals, not as a decimal: https://cloud.google.com/bigquery/docs/reference/standard-sql/datetime_functions#datetime_diff